lib/api/api.dart

```diff
@@ -28,7 +28,7 @@ abstract class AppApi {
   Future<AppListFunds?> listFunds();
 
   /// Return the list of invoices from lightning node.
-  Future<AppListInvoices> listInvoices();
+  Future<AppListInvoices?> listInvoices({required String status});
```

Comment:
```suggestion
  Future<AppListInvoices> listInvoices({required String status});
```

why a nullable value?